lib/launcher.js

```diff
@@ -47,7 +47,25 @@ class Launcher {
             return reporter
         }
 
+        const baseReporterOptions = config.reporterOptions || {}
+
         for (let reporterType of config.reporter) {
+            config.reporterOptions = {}
```

Comment:
I am afraid this will overwrite itself in the reporter will still be the value of `baseReporterOptions` after one event loop cycle since we passing an object per reference. So when the first event occurs the config in the reporter will still be the old value (not sure though). How about we do:

```js
let reporterOptions = Object.assign(config.reporterOptions);
// then delete other reporter options
```